projects/cas3-and-delius/deploy/values-dev.yml

```diff
@@ -9,6 +7,9 @@ generic-service:
 
   env:
     SENTRY_ENVIRONMENT: dev
+    SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_HMPPS-AUTH_TOKEN-URI: https://sign-in-dev.hmpps.service.justice.gov.uk/auth/oauth/token
```

Comment:
Do we need to add the approved-premises-api URL in here too?

Reply:
I didn't think so since we only use the url from the message?

Reply:
Ah good point sorry!